django/publicmapping/publicmapping/settings.py

```diff
@@ -37,7 +37,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = os.getenv('DEBUG', False)
 
-ALLOWED_HOSTS = ['web']
+ALLOWED_HOSTS = ['localhost']
```

Comment:
This only allows the site to be accessed at `http://localhost:8080`; if I access via `http://lima.internal.azavea.com:8080` (and eventually, when this gets deployed to staging), this causes / will cause a `DisallowedHost: Invalid HTTP_HOST header: 'lima.internal.azavea.com:8080'. You may need to add u'lima.internal.azavea.com' to ALLOWED_HOSTS.` exception to be raised. We'll need to be able to set this on a per-environment basis.